query_optimizer/tests/execution_generator/Select.test

```diff
@@ -656,7 +656,7 @@ SELECT SUM(EXTRACT(SECOND FROM value))
 FROM dates;
 --
 +-------------------------------+
-|SUM(EXTRACT(SECOND FROM value))|
+|SUM(EXTRACT(second FROM value))|
```

Comment:
Yes, I think the convention would be that all variables (i.e., table and attribute names) are lowercase, while all key words should be uppercase.